src/video/VideoPlayer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import YouTube from 'react-youtube';
+
+@connect(state => ({urls: state.urls, idx: state.currentIndex}))
```

Comment:
That part here is likely not working.
You need to write it as {videos: state.videos}